sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/Schema.java

```diff
@@ -986,10 +1246,8 @@ public Field getField(String name) {
   /** Find the index of a given field. */
   public int indexOf(String fieldName) {
     Integer index = fieldIndices.get(fieldName);
-    if (index == null) {
```

Comment:
This change caused BEAM-10631, we should always use if-blocks (as it was), or lazy `Preconditions.checkArgument(boolean condition, String format, Object args...)`.